pkg/handlers/v1/cloud_fetch.go

```diff
@@ -31,14 +31,15 @@ type PagedCloudAssets struct {
 
 // CloudAssetDetails represent an asset and associated attributes
 type CloudAssetDetails struct {
-	PrivateIPAddresses []string          `json:"privateIpAddresses"`
-	PublicIPAddresses  []string          `json:"publicIpAddresses"`
-	Hostnames          []string          `json:"hostnames"`
-	ResourceType       string            `json:"resourceType"`
-	AccountID          string            `json:"accountId"`
-	Region             string            `json:"region"`
-	ARN                string            `json:"arn"`
-	Tags               map[string]string `json:"tags"`
+	PrivateIPAddresses []string            `json:"privateIpAddresses"`
+	PublicIPAddresses  []string            `json:"publicIpAddresses"`
+	Hostnames          []string            `json:"hostnames"`
+	ResourceType       string              `json:"resourceType"`
+	AccountID          string              `json:"accountId"`
+	Region             string              `json:"region"`
+	ARN                string              `json:"arn"`
+	Tags               map[string]string   `json:"tags"`
+	AccountOwner       domain.AccountOwner `json:"accountOwner"`
```

Comment:
The api descriptor too (missing champions).

Reply:
+1 to this please fix @sydneyteh96

Reply:
Champions is a field in AccountOwner struct.

Reply:
got it!